tests/jsmarionette/client/sockit-to-me/package.json

```diff
@@ -6,7 +6,7 @@
   "main": "build/Release/sockit",
 
   "devDependencies": {
-    "node-gyp": "~0.10.6"
+    "node-gyp": "3.0.1"
```

Comment:
I also wouldn't tie sockit-to-me to that specific version of node-gyp, instead, look at the version shipped typically with 0.12 and use that as the base version.